index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Itim&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Itim&family=Lobster&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    
+</head>
+<body>
+    <header>
+        <h1>PANCAKES or CREPES or MLYNTSI</h1>
+    </header>
+    <main>
+        <img id="top_image" src="/images/Bez-nachinki.jpg" alt="pancakes on a plate"/>
+        <div id="ingredients">
+            <h3>You will need</h3>
+            <ul>
+                <li>4 eggs</li>
+                <li>1 glass of milk</li>
+                <li>1/2 teaspoon salt</li>
```

Comment:
have a look at this if you wanna use a html special character instead of `1/2` https://www.w3schools.com/charsets/ref_html_entities_4.asp